helm/coredns-app/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-name: coredns
+name: coredns-app
```

Comment:
Is this still part of reverting the previous update? This should be independent of the `coredns` version

Reply:
I think was was left over from my last testing. Thanks for pointing this out. That should not be there I'll fix it.